packages/sdk/src/actor/behaviors/index.ts

```diff
@@ -10,3 +10,5 @@ export * from './behavior';
 export * from './behaviorTypes';
 export * from './buttonBehavior';
 export * from './targetBehavior';
+export * from './toolBehavior';
+export * from './penBehavior';
```

Comment:
@tombuMS I'm getting a lint warning from this line. Can you clean it up when you have a moment?
```
C:\code\Microsoft\mixed-reality-extension-sdk\packages\sdk\src\actor\behaviors\index.ts
  14:31  warning  Newline required at end of file but not found  eol-last
```